Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:14.04
+MAINTAINER Yi Wang <yi.wang.2005@gmail.com>
+
+COPY build.sh /opt
+
+CMD /opt/build.sh
```

Comment:
is using build.sh instead of putting content of build.sh here to create fewer docker layers? Nice!